feat(login): Add login page

matiboy opened the pull request:
- Login page templates and url
- Change variable names for subject
- add countdown timer in
email
- update README

matiboy committed:
feat(login): Add login page

- Login page templates and url
- Change variable names for subject
- add countdown timer in
email
- update README

Jafnee commented on django_pin_auth/forms.py:
```diff
@@ -41,13 +41,13 @@ def prepare_form(self):
     def send_email(self, request):
```
Line 38 raises a User matching query does not exist. when logging in with unused email.

Jafnee replied:
Is that intentional, for the library user to catch?

matiboy replied:
No, it was meant to be caught at the form validation level.
Will fix

matiboy committed:
fix(verify): Login should not allow email that doesnt exist

- Declare clean email in LoginForm and check for existing email
- Corresponding tests
